RELEASE-NOTES.rst

```diff
@@ -2,7 +2,7 @@
  Invenio-Assets v1.0.0
 =======================
 
-Invenio-Assets v1.0.0 was released on December 4, 2015.
+Invenio-Assets v1.0.0 was released on June 29, 2016.
 
```

Comment:
Well, that may be a bit premature, since the future releases will be most probably tagged as `v1.0.0b1` or `v1.0.0rc1` still... Depends on how we organise the API cleanup and the documentation clean-up, which itself depends on the available manpower...